build.go

```diff
@@ -114,7 +114,7 @@ func nixBuild(drvs map[string]bool, buildArgs []string) error {
 			numChars = numBuildChars
 			args = buildArgs
 		}
-		args = append(args, drv)
+		args = append(args, drv+"^*")
```

Comment:
basically something like this is needed
```suggestion
                if nixVersion >= 2.15 then
		args = append(args, drv+"^*")
		else
		args = append(args, drv)
```